spec/amber/cli/commands_spec.cr

```diff
@@ -38,6 +38,8 @@ module Amber::CLI
         MainCommand.run ["new", TESTING_APP, "--deps"]
         Dir.cd(TESTING_APP)
         MainCommand.run ["generate", "scaffold", "Animal", "name:string"]
+		`rm ./shard.yml`
```

Comment:
Replace the generated shard.yml with a test shard to test the current amber context.

Reply:
@eliasjpr is there a way to just add the `path: ../amber` to the shard instead of replacing the whole file?  I'm concerned that we will forget to update this second shard.yml and it will get out of date.

Reply:
Use the `sed` command instead to replace just the line. Note that this is looking by line number.